node/rate_limiter.js

```diff
@@ -72,10 +73,16 @@ function RateLimiter(options) {
     }
     var self = this;
 
-    self.timers = options.timers;
+    // stats
+    self.channel = options.channel;
+    assert(self.channel && !self.channel.topChannel, 'RateLimiter requires top channel');
+    self.rateLimiterEmptyTags = new stat.RateLimiterEmptyTags();
```

Comment:
technically this is not safe to do; lets just create a new empty object every time.